azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java

```diff
@@ -39,36 +39,36 @@
     boolean isPrimary();
 
     /**
-     * @return the MAC Address of the network interface
+     * @return the Mac Address of the network interface
      */
     String macAddress();
 
     /**
      *
-     * @return the Internal DNS name assigned to this network interface
+     * @return the Internal Dns name assigned to this network interface
```

Comment:
Same here --- comments use the same caps as just plain English. Camel casing is used only in the code.